README.md

```diff
@@ -30,21 +30,235 @@
 
-[![PostgresML dashboard](pgml-docs/docs/images/dashboard/models.png)](https://cloud.postgresml.org/)
+### Text Data
+- Perform natural language processing (NLP) tasks like sentiment analysis, question and answering, translation, summarization and text generation
+- Access 1000s of state-of-the-art language models like GPT-2, GPT-J, GPT-Neo from :hugs: HuggingFace model hub
+- Fine tune large language models (LLMs) on your own text data for different tasks
 
-<h2 align="center">
-    See it in action — <a href="https://cloud.postgresml.org/" target="_blank">cloud.postgresml.org</a>
-</h2>
+**Translation**
+<table>
+<tr>
+<td>SQL Query</td>
+<td>Result </td>
+</tr>
+<tr>
+<td>
+
+```sql
+SELECT pgml.transform(
+    'translation_en_to_fr',
+    inputs => ARRAY[
+        'Welcome to the future!',
+        'Where have you been all this time?'
+    ]
+) AS french;
+```
+</td>
+<td>
+
+```sql
+                         french                                 
+------------------------------------------------------------
+
+[
+    {"translation_text": "Bienvenue à l'avenir!"},
+    {"translation_text": "Où êtes-vous allé tout ce temps?"}
+]
+```
+</td>
+</tr>
+</table>
+
+
+
+**Sentiment Analysis**
+<table>
+<tr>
+<td>SQL Query</td>
+<td>Result </td>
+</tr>
+<tr>
+<td>
+
+```sql
+SELECT pgml.transform(
+
+    '{"model": "roberta-large-mnli"}'::JSONB,
+    inputs => ARRAY
+    [
+        'I love how amazingly simple ML has become!', 
+        'I hate doing mundane and thankless tasks. ☹️'
+    ]
+
+) AS positivity;
+```
+</td>
+<td>
+
+```sql
+                    positivity
+------------------------------------------------------
+[
+    {"label": "NEUTRAL", "score": 0.8143417835235596}, 
+    {"label": "NEUTRAL", "score": 0.7637073993682861}
+]
+```
+</td>
+</tr>
+</table>
+
+
+### Tabular data
+- [47+ classification and regression algorithms](https://postgresml.org/docs/guides/training/algorithm_selection)
+- [8 - 40X faster inference than HTTP based model serving](https://postgresml.org/blog/postgresml-is-8x-faster-than-python-http-microservices)
+- [Millions of transactions per second](https://postgresml.org/blog/scaling-postgresml-to-one-million-requests-per-second)
+- [Horizontal scalability](https://github.com/postgresml/pgcat)
+
+
+**Training a classification model**
+
+<table>
+<tr>
+<td> Training </td>
+<td> Inference </td>
+</tr>
+<tr>
+<td>
+
+
+```sql
+SELECT * FROM pgml.train(
+    'Handwritten Digit Image Classifier',
+    algorithm => 'xgboost',
+    'classification',
+    'pgml.digits',
+    'target'
+);
+```
+
+</td>
+<td>
+
+```sql
+SELECT pgml.predict(
+    'My Classification Project', 
+    ARRAY[0.1, 2.0, 5.0]
+) AS prediction;
+```
+</td>
+</tr>
+</table>
+
+## Installation
```

Comment:
Should this be a deployment rather than installation? I dunno...

Reply:
We could use Deployment for production deployments and provide guidelines.